docs/scalardb-server.md

```diff
@@ -0,0 +1,102 @@
+# Scalar DB server
+
+Scalar DB server is a gRPC server that implements Scalar DB interface. 
+With Scalar DB server, you can use Scalar DB features from multiple programming languages that are supported by gRPC.
+
+Currently, we provide only a Java client officially, and we will support other language clients officially in the future.
+Of course, you can generate language-specific client stubs by yourself.
+However, note that it might not be straightforward to implement a client since it's using a bidirectional streaming RPC in gRPC, and you need to be familiar with it.
```

Comment:
```suggestion
However, note that it is not necessarily straightforward to implement a client since it's using a bidirectional streaming RPC in gRPC, and you need to be familiar with it.
```